Tests/Microsoft.Teams.Cards.Tests/CardTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+using System.Text.Json;
```

Comment:
I think we can get rid of the cards tests since David auto generated us our code now. Sorry for the last minute change in direction on that, otherwise the Api package tests look great!